src/main/webapp/loggedin/logout.js

```diff
@@ -0,0 +1,30 @@
+//adds username to each page
+document.addEventListener('DOMContentLoaded', (event) => {
+    var usernameElem = document.getElementById("username");
+    if(usernameElem){
+        usernameElem.innerHTML = localStorage.getItem("username");
+    }
+});
+ 
+//signs out of the page
+function signOut() {
+    //log out page --> DOESNT RUN FOR SOME REASON??
+ 
+    //log out google
+  gapi.load('auth2', function() {
+       gapi.auth2.init({
+         client_id: '610251294652-9ojdhjhh8kpcdvdbmrvp9nkevgr2n2d8.apps.googleusercontent.com'
```

Comment:
I don't think we want this key to be stored here...

Reply:
oh shoot yea ur right